1_fetch/src/get_nwis_data.R

```diff
@@ -46,3 +45,12 @@ download_nwis_site_data <- function(filepath, parameterCd = '00010', startDate="
   return(filepath)
 }
 
+combine_nwis_site_data <- function(files_in, file_out){
+  ts_list <- list()
+  for (f in files_in){
+    ts_list[[f]] <- read.csv(f, colClasses = c(rep("character", 3), "numeric", rep("character", 2)))
+  }
+  ts_df <- do.call("rbind", ts_list)
+  write.csv(ts_df, file = file_out, row.names=FALSE)
+  return(file_out)
+}
```

Comment:
Excellent! I love seeing how people solve the read and bind problem since there are so many ways to do it. I'm a fan of the `purrr` package to do this (though I always have to look it up because the syntax is not in my memory yet):

`ts_df <- files_in %>% purrr::map_dfr(read.csv)`